validator.bats

```diff
@@ -609,17 +619,6 @@ LUM-2345'
   [[ "$status" -eq $ERROR_SUBJECT ]]
 }
 
-@test "overall validation invalid body length" {
-  MESSAGE='feat(scope1): subject
-
-12345678901234567890123456789012345678901234567890123456789012345678901234567890123456789012345678901
```

Comment:
Modify this one instead to test with "regular" space-separated words, we still want to check that the validator forbids long lines in other cases